index.html

```diff
@@ -46,36 +66,37 @@ <h2>I’am a Software Developer</h2>
             </li>
           </ul>
         </div>
+      
     </section>
     </div>
     <section class="grid-work">
-      <div class="work-sec">
-      <div class="work-heading"><h1>My Recent Works</h1>
+      <!---->
+      <div class="work-heading"><h2>My Recent Works</h2>
       <hr>
       </div>
       <div class="photo-work">
-        <img src="./Pictures/ImgPlaceholder.png" alt="Work Photo">
-         
-        <div class="photo-text">
-          <h1>Multi-Post Stories</h1>
-          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+        <div class="card-1">
+       <div class="work-sec"> <img src="./Pictures/ImgPlaceholder.png" alt="Work Photo"></div>
+         <div class="photo-descrip">
+          <h1 class="photo-text">Multi-Post Stories</h1>
+          <p class="photo-para">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
           </p>
-        </div>
-     
-      <div class="work-list">
-        <ul>
+        <ul class="work-list">
           <a href="#"><li class="item work-item-1">css</li></a>
           <a href="#"><li class="item work-item-2">html</li></a>
           <a href="#"><li class="item work-item-3">bootstrap</li></a>
           <a href="#"><li class="item work-item-4">Ruby</li></a>
         </ul>
-      </div>
       <div class="link">
         <a href="#">See Project</a>
       </div>

```

Comment:
- [ ] Kindly consider adding appropriate padding and margin to match your design like the one in the Figma

| Your design  | Figma design|
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/53402600/200132690-73a5bd4f-a841-44bf-b719-87fbaf76ff57.png) | ![image](https://user-images.githubusercontent.com/53402600/200132678-6389136b-5998-48e5-9c76-d0a43685f6b6.png)|